src/extension.ts

```diff
@@ -398,3 +410,15 @@ async function runJavaProject(node: any, noDebug: boolean) {
         throw ex;
     }
 }
+
+function findLaunchConfiguration(mainClass: string, projectName: string | undefined, filePath?: string): vscode.DebugConfiguration | undefined {
+    const workspaceFolder: vscode.WorkspaceFolder | undefined =
+            filePath ? vscode.workspace.getWorkspaceFolder(vscode.Uri.file(filePath)) : undefined;
+    const launchConfigurations: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration("launch", workspaceFolder);
+    const existingConfigs: vscode.DebugConfiguration[] = launchConfigurations.configurations;
+    const existConfig: vscode.DebugConfiguration | undefined = _.find(existingConfigs, (config) => {
```

Comment:
What if the user has two debug configurations, whose `mainClass` and `projectName` are the same but other options like args, vmArgs are different?

Reply:
The first one will be used. This might be a rare case, since no complaint received yet.